openhcl/underhill_attestation/src/secure_key_release.rs

```diff
@@ -164,7 +164,12 @@ pub async fn request_vmgs_encryption_keys(
             }) => {
                 tracing::warn!(CVM_ALLOWED, retry = i, "Failed to get VMGS key-encryption")
             }
-            Err(e) if i == (max_retry - 1) => Err(e)?,
+            Err(e) if i == (max_retry - 1) => {
+                tracing::error!(
```

Comment:
do you not want to include e as well here?

Reply:
Good point.